dotcom-rendering/src/lib/content.d.ts

```diff
@@ -625,7 +626,9 @@ type Weighting =
 	| 'showcase'
 	| 'halfwidth'
 	| 'immersive'
-	| 'richLink'; // Note, 'richLink' is used internally but does not exist upstream.
+	| 'richLink' // Note, 'richLink', 'inlineBox' and 'inlineBoxClearAtAllBreakpoints' are used internally but do not exist upstream.
+	| 'inlineBox'
```

Comment:
Could needing to do this be avoided by making sure we always insert the embed between to <p> tags?

Reply:
I wasn't aware there were external constraints on those values. I had assumed the Figures were the intended place for css related to the layout of the contentElement, but having the  css on the `EmailSignup` probably makes more sense.

The PR description goes through the float behaviour this was meant to address, but that's not clear in the code. If we move the styling fix  this on to the `EmailSignup` component, it wont' be necessary.

The placement logic does try to find a space away from floating elements and after a `<p>` but we can't be sure there will always be a suitable space, so we also need a css fix. Since the floating element can be taller than the element that come after it, there might need to be more than one `<p>` between the the floating element and the embed

Reply:
made those changes - the styling is now on `EmailSignup`